ouroboros-network/src/Ouroboros/Network/Testing/ConcreteBlock.hs

```diff
@@ -209,29 +209,31 @@ mkChain =
 mkChainSimple :: [BlockBody] -> Chain Block
 mkChainSimple = mkChain . zip [1..]
 
--- TODO: this generator will generate only chain fragments which start with
--- @BlockNo 0@ and @GenesisHash@ in @prevHeaderHash@.
-mkChainFragment :: [(SlotNo, BlockBody)] -> ChainFragment Block
-mkChainFragment =
-    fixupChainFragmentFromGenesis fixupBlock
+mkChainFragment :: ChainHash Block
+                -> BlockNo
+                -> [(SlotNo, BlockBody)]
+                -> ChainFragment Block
+mkChainFragment anchorhash anchorblockno =
```

Comment:
Nitpicking: why are you not using camelCase for these arguments? Not just here, but throughout this PR.

Reply:
We have things like `blockNo` as functions, but we also sometimes want to use a `blockno` as a local var.